ubuntudesign/www.ubuntu.com
# Remove unmentioned template includes

The number of old and unused template files polluting our repository has been bugging me for a while.

I ran [this script](https://gist.github.com/nottrobin/dc4a147260d9089405b1e2f920a06d96) in the `templates` directory to delete them. Don't worry, they'll still be in version history.

QA
--

```
make run
```

Browse around the whole site and check pages still look fine and as they do on live. Especially footers, contextual navs, sub navs, breadcrumbs etc.
